plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -211,7 +212,7 @@ func (p *Plugin) computeSecurityContext(a admission.Attributes, pod *api.Pod, sp
 	klog.V(4).Infof("getting pod security policies for pod %s (generate: %s)", pod.Name, pod.GenerateName)
 	var saInfo user.Info
 	if len(pod.Spec.ServiceAccountName) > 0 {
-		saInfo = serviceaccount.UserInfo(a.GetNamespace(), pod.Spec.ServiceAccountName, "")
+		saInfo = serviceaccount.UserInfo(a.GetNamespace(), pod.Spec.ServiceAccountName, "", pod.Tenant)
```

Comment:
shall the tenant value be obtained from admission attributes or the pod? Not quite sure about the difference, yet I see that the namespace value is from the admission attributes.

Reply:
fixed